readthedocs/projects/models.py

```diff
@@ -195,6 +195,11 @@ class AddonsConfig(TimeStampedModel):
 
     # File Tree Diff
     filetreediff_enabled = models.BooleanField(default=False, null=True, blank=True)
+    filetreediff_ignored_files = models.JSONField(
```

Comment:
This could also just be a textfield, don't think we need a jsonfield, but also I don't see the downsides.

Reply:
I used JSON because it could be useful if we need to perform a query over the field --being a blob of text will make that more difficult.